react/features/toolbox/components/web/Toolbox.js

```diff
@@ -1047,14 +1017,36 @@ class Toolbox extends Component<Props, State> {
                     key = 'fullscreen'
                     onClick = { this._onToolbarToggleFullScreen }
                     text = { _fullScreen ? t('toolbar.exitFullScreen') : t('toolbar.enterFullScreen') } />,
-            this._shouldShowButton('livestreaming')
-                && <LiveStreamButton
-                    key = 'livestreaming'
-                    showLabel = { true } />,
+            this._shouldShowButton('security')
+            && <SecurityDialogButton
+                key = 'security'
+                showLabel = { true } />,
+            this._shouldShowButton('closedcaptions')
+            && <ClosedCaptionButton
+                key = 'closed-captions'
+                showLabel = { true } />,
             this._shouldShowButton('recording')
                 && <RecordButton
                     key = 'record'
                     showLabel = { true } />,
+            this._shouldShowButton('localrecording')
+                && <OverflowMenuItem
+                    accessibilityLabel = { t('toolbar.accessibilityLabel.localRecording') }
+                    icon = { IconRec }
+                    key = 'localrecording'
+                    onClick = { this._onToolbarOpenLocalRecordingInfoDialog }
+                    text = { t('localRecording.dialogTitle') } />,
+            this._shouldShowButton('mute-everyone')
+                && <MuteEveryoneButton
+                    key = 'mute-everyone'
+                    showLabel = { true } />,
+            this._shouldShowButton('livestreaming')
+                && <LiveStreamButton
+                    key = 'livestreaming'
+                    showLabel = { true } />,
+            <hr
```

Comment:
If, for some weird reason, several of these buttons were disabled, wouldn't the separators be shown on top of one another? I was wondering, if it's not too complicated, if we had a way to discriminate whether we should show these or not. (Similarly to the profile one)

Reply:
Fixed it.